src/routes/healthz.rs

```diff
@@ -0,0 +1,4 @@
+#[tracing::instrument]
+pub async fn healthz() -> &'static str {
```

Comment:
Why healthz?

Reply:
Just a convention I have adopted from previous use, largely inspired by kubernetes

https://kubernetes.io/docs/reference/using-api/health-checks/

Changed for brevity though